daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md

```diff
@@ -99,23 +99,11 @@ def process_batch():
 The `batch-sdk` service uses the PostgreSQL output binding defined in the [`binding-postgres.yaml`]({{< ref "#componentbinding-postgresyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table. 
 
 ```python
-with DaprClient() as d:
-    sqlCmd = ('insert into orders (orderid, customer, price) values' +
-              '(%s, \'%s\', %s)' % (order_line['orderid'],
-                                    order_line['customer'],
-                                    order_line['price']))
-    payload = {'sql': sqlCmd}
-
-    print(sqlCmd, flush=True)
-
-    try:
-        # Insert order using Dapr output binding via HTTP Post
-        resp = d.invoke_binding(binding_name=sql_binding, operation='exec',
-                                binding_metadata=payload, data='')
-        return resp
-    except Exception as e:
-        print(e, flush=True)
-        raise SystemExit(e)
+try:
+    # Insert order using Dapr output binding via HTTP Post
+    resp = d.invoke_binding(binding_name=sql_binding, operation='exec',
```

Comment:
I like it that you've shortened.  however I think the line with `With DaprClient as d:` is still needed to make sense of the `d.invoke_binding` line.  Also for bindings the metadata/payload is vital, so I think you need all the lines from d's declaration to the invoke_binding, inclusive of the SQL command and the payload..